src/openms_gui/source/VISUAL/TOPPASVertex.cpp

```diff
@@ -77,20 +78,19 @@ namespace OpenMS
   QStringList TOPPASVertex::TOPPASFilenames::getSuffixCounts() const
   {
     // display file type(s)
-    std::map<QString, Size> suffices;
-    for (const QString& fn : filenames_)
+    std::map<String, Size> suffices;
+    for (const auto& fn : filenames_)
     {
-      QStringList l = QFileInfo(fn).completeSuffix().split('.');
-      QString suf = ((l.size() > 1 && l[l.size() - 2].size() <= 4) ? l[l.size() - 2] + "." : QString()) + l.back(); // take up to two dots as suffix (the first only if its <=4 chars, e.g. we want ".prot.xml" or ".tar.gz", but not "stupid.filename.with.longdots.mzML")
-      ++suffices[suf];
+      auto type = FileHandler::getType(fn.toStdString());
```

Comment:
@poshul was this the code you had to change before?

Reply:
I would be surprised if this code is problematic.
@poshul ?